[webpack-config] node-progress dependency missing

## Description

Running `next build` within Expo project with `next-adapter` on a CI env fails.
Note I first run yarn install in production mode.

## Expected Behavior

Running `next build` within Expo project with `next-adapter` on a CI env should not fail.

## Observed Behavior

```
yarn run v1.22.0
$ next build
> Build error occurred
Error: Cannot find module 'progress'
Require stack:
- /home/vsts/work/1/s/node_modules/@expo/webpack-config/webpack/plugins/ExpoProgressBarPlugin.js
- /home/vsts/work/1/s/node_modules/@expo/webpack-config/webpack/plugins/index.js
- /home/vsts/work/1/s/node_modules/@expo/webpack-config/webpack/addons/withUnimodules.js
- /home/vsts/work/1/s/node_modules/@expo/webpack-config/webpack/addons/index.js
- /home/vsts/work/1/s/node_modules/@expo/webpack-config/addons/index.js
- /home/vsts/work/1/s/node_modules/@expo/next-adapter/build/withExpo.js
- /home/vsts/work/1/s/node_modules/@expo/next-adapter/build/index.js
- /home/vsts/work/1/s/packages/PublishingManager/next.config.js
- /home/vsts/work/1/s/node_modules/next/dist/next-server/server/config.js
- /home/vsts/work/1/s/node_modules/next/dist/build/index.js
- /home/vsts/work/1/s/node_modules/next/dist/cli/next-build.js
- /home/vsts/work/1/s/node_modules/next/dist/bin/next
```

I suspect the issue is due to the [progress package](https://github.com/visionmedia/node-progress) required [here](https://github.com/expo/expo-cli/blob/master/packages/webpack-config/src/plugins/ExpoProgressBarPlugin.ts#L2) not being listed as dependency in webpack-config [package.json](https://github.com/expo/expo-cli/blob/master/packages/webpack-config/package.json#L38) (note progress is a [dependency](https://github.com/clessg/progress-bar-webpack-plugin/blob/master/package.json#L27) of `progress-bar-webpack-plugin` which in turn it is a [dependency](https://github.com/expo/expo-cli/blob/master/packages/webpack-config/package.json#L59) of webpack-config).

--- a/packages/webpack-config/package.json
+++ b/packages/webpack-config/package.json
@@ -55,7 +55,7 @@
     "optimize-css-assets-webpack-plugin": "^5.0.1",
     "pnp-webpack-plugin": "^1.5.0",
     "postcss-safe-parser": "^4.0.1",
-    "progress-bar-webpack-plugin": "^1.12.1",
+    "progress": "^2.0.3",
     "react-dev-utils": "9.0.3",
     "style-loader": "^0.23.1",
     "terser-webpack-plugin": "^1.2.3",

